src/components/portfolio/DetailsMainBacktestingList.tsx

```diff
@@ -0,0 +1,226 @@
+            </ListItemText>
+          </AccordionSummary>
+          <AccordionDetails className={classes.labelText}>
+            Description placeholder
+          </AccordionDetails>
+        </Accordion>
+      </ListItem>
+      <ListItem>
+        <Accordion className={classes.accordion}>
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon className={classes.expandButton} />}
+            aria-controls="panel1a-content"
+            id="panel1a-header"
+          >
+            <ListItemIcon className={classes.listIcon}>
+              <WarningIcon className={classes.beta} />
+            </ListItemIcon>
+            <ListItemText className={classes.labelText}>
+              {t('portfolio.details.backtesting.standardDeviation')}
+            </ListItemText>
+            <ListItemText className={classes.valueText}>
+              <Performance value={standardDeviation} size="1em" noPaint />
+            </ListItemText>
+          </AccordionSummary>
+          <AccordionDetails className={classes.labelText}>
+            Description placeholder
+          </AccordionDetails>
+        </Accordion>
+      </ListItem>
+      <ListItem>
+        <Accordion className={classes.accordion}>
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon className={classes.expandButton} />}
+            aria-controls="panel1a-content"
+            id="panel1a-header"
+          >
+            <ListItemIcon className={classes.listIcon}>
+              <NewReleasesIcon className={classes.sharpe} />
+            </ListItemIcon>
+            <ListItemText className={classes.labelText}>
+              {t('portfolio.details.backtesting.sharpeRatio')}
+            </ListItemText>
+            <ListItemText className={classes.valueText}>
+              {sharpeRatio}
+            </ListItemText>
+          </AccordionSummary>
+          <AccordionDetails className={classes.labelText}>
+            Description placeholder
+          </AccordionDetails>
+        </Accordion>
+      </ListItem>
+      <ListItem>
+        <Accordion className={classes.accordion}>
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon className={classes.expandButton} />}
+            aria-controls="panel1a-content"
+            id="panel1a-header"
+          >
+            <ListItemIcon className={classes.listIcon}>
+              <SpeedIcon className={classes.cagr} />
+            </ListItemIcon>
+            <ListItemText className={classes.labelText}>
+              {t('portfolio.details.backtesting.cagr')}
+            </ListItemText>
+            <ListItemText className={classes.valueText}>
+              <Performance value={cagr} size="1em" noPaint />
+            </ListItemText>
+          </AccordionSummary>
+          <AccordionDetails className={classes.labelText}>
+            Description placeholder
+          </AccordionDetails>
+        </Accordion>
+      </ListItem>
```

Comment:
Each of these items is basically the same, just the icon and texts are different. Wouldn't it make sense to create a component?